vita3k/gui/src/gui.cpp

```diff
@@ -63,19 +63,11 @@ void draw_info_message(GuiState &gui, EmuEnvState &emuenv) {
         ImGui::SetNextWindowPos(ImVec2(emuenv.viewport_pos.x + (display_size.x / 2) - (WINDOW_SIZE.x / 2.f), emuenv.viewport_pos.y + (display_size.y / 2.f) - (WINDOW_SIZE.y / 2.f)), ImGuiCond_Always);
         ImGui::PushStyleVar(ImGuiStyleVar_ChildRounding, 10.f * SCALE.x);
         ImGui::BeginChild("##info", WINDOW_SIZE, true, ImGuiWindowFlags_NoSavedSettings | ImGuiWindowFlags_NoDecoration);
-        auto title = fmt::format("{}", spdlog::level::to_string_view(gui.info_message.level));
-        title[0] = std::toupper(title[0]);
-        ImGui::SetWindowFontScale(RES_SCALE.x);
-        ImGui::SetCursorPosX((ImGui::GetWindowWidth() - ImGui::CalcTextSize(title.c_str()).x) / 2);
-        ImGui::TextColored(GUI_COLOR_TEXT_TITLE, "%s", title.c_str());
-        ImGui::Spacing();
-        ImGui::Separator();
```

Comment:
Is there a reason for deleting this? Because it can't be translated?

Reply:
It looks more aesthetically pleasing, i did not know right or not

![image](https://github.com/Vita3K/Vita3K/assets/61804715/bfbdc2b0-244c-43af-8a42-449c77d5d22f)

Reply:
Please, don't delete this, I'm put here for a good reason, it's to indicate whether it's an error or information